ArcadiaTeamsBot.CQRS/Handlers/GetServiceDeskRequestPrioritiesHandler.cs

```diff
@@ -0,0 +1,36 @@
+﻿using ArcadiaTeamsBot.CQRS.Queries;
+using MediatR;
+using Newtonsoft.Json;
+using ServiceDesk.Abstractions.DTOs;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ArcadiaTeamsBot.CQRS.Handlers
+{
+    class GetServiceDeskRequestPrioritiesHandler : IRequestHandler<GetServiceDeskRequestPrioritiesQuery, IEnumerable<ServiceDeskRequestPriorityDTO>>
+    {
+        public async Task<IEnumerable<ServiceDeskRequestPriorityDTO>> Handle(GetServiceDeskRequestPrioritiesQuery request, CancellationToken cancellationToken)
+        {
+            var prioritiesJsonAsString = 
+                @"[
+                        {
+                        'key': 1,
+                        'value': 'Low'
+                        },
+                        {
+                        'key': 2,
+                        'value': 'Default'
+                        },
+                        {
+                        'key': 3,
+                        'value': 'High'
+                        }
+                ]";
+
+            var serviceDeskRequestPriorities = JsonConvert.DeserializeObject<IEnumerable<ServiceDeskRequestPriorityDTO>>(prioritiesJsonAsString);
```

Comment:
Couldn't you implemented something... simpler? For example:
```cs
private ServiceDeskPriorityDTO[] priorities = new [] {
  new ServiceDeskPriorityDTO
  {
    ...
  },
  ...
}
```
instead of deserialize same raw json string to objects each time the method is called.

Reply:
But in the future it will be the response in JSON format. We will need just replace temporary string with getting response.

Reply:
Ok. Made it a temporary class field

Reply:
If in future it is loaded another way - you'll need to rewrite almost whole class. Class field will be the last thing you'll need to worry about.